.github/workflows/publish.yml

```diff
@@ -9,7 +9,6 @@ on:
 
 jobs:
   publish:
-    name: Publish
```

Comment:
Why remove?

Reply:
Just for consistency sake.

The other ones use an uppercase workflow name and a lowercase job name.

Naming is hard 🤷‍♂

Reply:
I reverted the name 👍